src/easydmp/plan/management/commands/move_answer_data_to_answerset.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+from django.core.management import BaseCommand
+
+from easydmp.dmpt.models import Question
+from easydmp.plan.models import Plan, AnswerSet
+
+#
+# This was part of a refactor to change ownership of answer data, and may be deleted later.
+#
+
+def handle_plan(plan: Plan, delete: bool):
+    for k, d in dict(plan.data).items():
+        question = Question.objects.get(pk=int(k))
+        answerset = AnswerSet.objects.get(section=question.section)
+        answerset.data[k] = d
+        answerset.save()
+        if delete:
+            del plan.data[k]
+            plan.save()
+
+class Command(BaseCommand):
+    help = "Distribute answer data from Plan to appropriate AnswerSet"
```

Comment:
I'd mention "idempotent" here somehow since it is safe to rerun if `-d` is not used.